validator/specs/lib/validateB3dmSpec.js

```diff
@@ -1,9 +1,27 @@
 'use strict';
+var Cesium = require('cesium');
 var validateB3dm = require('../../lib/validateB3dm');
 
+var loadJson = Cesium.loadJson;
+
 describe('validateB3dm', function() {
+    var batchSchema;
+    beforeAll(function(done) {
+        console.log("beforeAll");
 
+        loadJson('https://raw.githubusercontent.com/AnalyticalGraphicsInc/3d-tiles/master/schema/batchTable.schema.json')
+            .then(function(schema) {
```

Comment:
Hm, `XMLHttpRequest` must only be supported in browsers and not Node. You could try looking at [http.request](https://nodejs.org/api/http.html#http_http_request_options_callback).